<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CS2MenuClass</class>
 <widget class="QMainWindow" name="CS2MenuClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>313</width>
    <height>291</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>313</width>
    <height>291</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>313</width>
    <height>291</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CS2Menu v0.0.1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>CS2 External Hacks ([INSERT] for show and hide)</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>180</y>
         <width>241</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>CS2 External Multihack by un4ckn0wl3z</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>30</y>
         <width>79</width>
         <height>100</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QCheckBox" name="distanceCheckBox">
          <property name="text">
           <string>Distance</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="aimbotCheckBox">
          <property name="text">
           <string>Aimbot</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="TriggerBotCheckBox">
          <property name="text">
           <string>TriggerBot</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="crosshairCheckBox">
          <property name="text">
           <string>Crosshair</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>95</width>
         <height>126</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="espBoxCheckBox">
          <property name="text">
           <string>ESP Box</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="healthBarCheckBox">
          <property name="text">
           <string>Health Bar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="healthStatusCheckBox">
          <property name="text">
           <string>Health Status</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="linesCheckBox">
          <property name="text">
           <string>Lines</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="teamCheckBox">
          <property name="text">
           <string>Team</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>313</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_CS2Menu"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout_CS2Menu">
   <property name="text">
    <string>About CS2Menu</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="CS2Menu.qrc"/>
 </resources>
 <connections/>
</ui>
